### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56073]
#### VFS team name
Benefits Team 1

#### VFS product name
Supplemental Claims Form 20-0995

#### VFS feature name
Supplemental Claims Form 20-0995

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
On the upload a document page, the button has the text "Upload"

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Follow the [Files pattern](https://design.va.gov/patterns/ask-users-for/files#default-state) and change text to "Upload a document"

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jaredcunha** [comment]
@shiragoodman The issue you've uncovered would be an issue with Platform code. The default text for file upload component is "Upload" so we should also make the change occur there as well to fix other instances on VA.gov.

**Comment by jaredcunha** [comment]
@shiragoodman Before proceeding with that, I want to understand what the source of truth is for the default state of the File Input. Is the image cited in this ticket up to date and accurate?

[Further down the File Input page](https://design.va.gov/patterns/ask-users-for/files#error-state), showing an error message, the button label is "Upload file" which I believe is an example of the `va-file-input` web component.
<img width="802" alt="Screenshot 2023-04-10 at 6 00 26 PM" src="https://user-images.githubusercontent.com/1094951/231007268-fe320451-1f95-4437-85c5-15a84b2b0eaf.png">

The `VaFileInput` component uses "Upload file" as the default text
<img width="913" alt="Screenshot 2023-04-10 at 6 01 32 PM" src="https://user-images.githubusercontent.com/1094951/231007484-73c534b4-a20b-4946-85b2-ce748c0c85b1.png">

As I mentioned about, the default button label for the Form System's `FileField` component is "Upload." This forms system uses a hardcoded file input instead of the design system component. 

So any follow-up work can be described accurately, please let us know what is the default in the event the screenshot in the design system had fallen out of date.

**Comment by eileen-coforma** [comment]
Exactly, but `Upload file` no capital on the f

**Comment by jaredcunha** [comment]
☝️ @allison0034  

We are going to change the text from `Upload a document` to `Upload file`. With that decision made, there are two other items that need to be done:

1. Update the [screen shots](https://design.va.gov/patterns/ask-users-for/files#default-state) on design.va.gov for the file upload pattern to align with the [content considerations](https://design.va.gov/patterns/ask-users-for/files#content-considerations). This is the current screenshot from design.va.gov.
![file-upload-input](https://user-images.githubusercontent.com/1094951/231251324-e88775a0-5806-47af-be6b-6e2150d6a914.png)

2. The form system default label is `Upload` and should be changed to `Upload file`.

**Comment by allison0034** [comment]
@humancompanion-usds are you ok with the changes above? As he points out our content style guide has "file"

**Comment by humancompanion-usds** [comment]
Yes, I'm great with it. I wrote those content considerations fairly recently (I think when we converted File input to a web-component back in October) and that's what Danielle and Beth prefer. 

@jaredcunha - Can you add a screenshot here once you've made that change so that I can update https://design.va.gov/patterns/ask-users-for/files#default-state ?

We're in the process of bringing web-components into the Forms library but this would be good to change in the existing widget (or wherever it is in the library). I have [filed a bug](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/188).

**Comment by eileen-coforma** [comment]
Staging still says `Upload a document` instead of `Upload file`

**Comment by rocio-desantiago** [comment]
@eileen-coforma ready

**Comment by eileen-coforma** [comment]
@rocio-desantiago Awesome. Looks good. Thanks!

@humancompanion-usds Here are two screenshots with the updated button language. (CC: @jaredcunha) 

Default:
![Screenshot 2023-04-13 at 2 44 29 PM](https://user-images.githubusercontent.com/111457818/231854908-e757b5fb-5a89-46b9-8dc2-83fb2d12a9d7.png)
Review state:
![Screenshot 2023-04-13 at 2 45 37 PM](https://user-images.githubusercontent.com/111457818/231855010-61149f94-64a6-4b0a-9df5-2378111dfe07.png)
